### PR TITLE
Added Quicksand V2.001

### PR DESCRIPTION
**Pull request by bghryct**
Taken from upstream repo https://github.com/bghryct/fonts.git at commit https://github.com/bghryct/fonts/commit/b3ba3ce81a869bfea74fbcd4bb0bc8a28ffdd183

### CODE REVIEW

**Comment by m4rc1e** [comment]
Can you reword your pr so it matches #2049?

We want to know the upstream repo is and at what commit the fonts come from.

**Comment by m4rc1e** [comment]
In your terminal, type in git log. You'll see how we structure our commit messages as well.

File: ofl/quicksand/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -3,4 +3,5 @@
 The project was initiated by Andrew Paglinawan in 2008 using geometric shapes as a core foundation. 
 It is designed for display purposes but kept legible enough to use in small sizes as well. 
 In 2016, in collaboration with Andrew, it was thoroughly revised by Thomas Jockin to improve the quality. 
+In 2019 Mirko Velimirovic made Quicksand Variable along the weight axis.
```
Comment (m4rc1e) [inline ofl/quicksand/DESCRIPTION.en_us.html]: Would prefer "In 2019, Mirko Velimirovic converted the family into a variable font."

**Comment by m4rc1e** [comment]
Pretty good overall. I just pulled your pr and changed the filename to Quicksand-VF.ttf so I could get FB results. 

## Fontbakery report

Fontbakery version: 0.7.8

<details>
<summary><b>[7] Quicksand-VF.ttf</b></summary>
<details>
<summary>🔥 <b>FAIL:</b> Checking file is named canonically.</summary>

* [com.google.fonts/check/canonical_filename](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/canonical_filename)
* 🔥 **FAIL** The file 'Quicksand-VF.ttf' must be renamed to 'QuicksandLight[wght].ttf' according to the Google Fonts naming policy for variable fonts.

</details>
<details>
<summary>🔥 <b>FAIL:</b> Does DESCRIPTION file contain a upstream Git repo URL?</summary>

* [com.google.fonts/check/description/git_url](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/description/git_url)
* 🔥 **FAIL** Please host your font project on a public Git repo (such as GitHub or GitLab) and place a link in the DESCRIPTION.en_us.html file.

</details>
<details>
<summary>🔥 <b>FAIL:</b> Does DESCRIPTION file mention when a family is available as variable font?</summary>

* [com.google.fonts/check/description/variable_font](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/description/variable_font)
* 🔥 **FAIL** Please include the following sentence in the DESCRIPTION.en-us.html file:

This family is available as a variable font.

</details>
<details>
<summary>⚠ <b>WARN:</b> Stricter unitsPerEm criteria for Google Fonts. </summary>

* [com.google.fonts/check/unitsperem_strict](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/unitsperem_strict)
* ⚠ **WARN** Even though unitsPerEm (1000) in this font is reasonable. It is strongly advised to consider changing it to 2000, since it will likely improve the quality of Variable Fonts by avoiding excessive rounding of coordinates on interpolations.

</details>
<details>
<summary>⚠ <b>WARN:</b> Are there caret positions declared for every ligature?</summary>

* [com.google.fonts/check/ligature_carets](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/ligature_carets)
* ⚠ **WARN** This font lacks caret position values for ligature glyphs on its GDEF table. [code: lacks-caret-pos]

</details>
<details>
<summary>⚠ <b>WARN:</b> Is there kerning info for non-ligated sequences?</summary>

* [com.google.fonts/check/kerning_for_non_ligated_sequences](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/kerning_for_non_ligated_sequences)
* ⚠ **WARN** GPOS table lacks kerning info for the following non-ligated sequences:
	- f + f
	- f + i
	- i + f
	- f + l
	- l + f
	- i + l

   [code: lacks-kern-info]

</details>
<details>
<summary>⚠ <b>WARN:</b> Name table strings must not contain the string 'Reserved Font Name'.</summary>

* [com.google.fonts/check/name/rfn](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/name.html#com.google.fonts/check/name/rfn)
* ⚠ **WARN** Name table entry ("Copyright 2019 The Quicksand Project Authors (https://github.com/andrew-paglinawan/QuicksandFamily.git), with Reserved Font Name "Quicksand"") contains "Reserved Font Name". This is an error except in a few specific rare cases.

</details>
<br>
</details>

### Summary

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | ℹ INFO | 🍞 PASS |
|:-----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 3 | 4 | 49 | 7 | 98 |
| 0% | 2% | 2% | 30% | 4% | 61% |

**Note:** The following loglevels were omitted in this report:
* **SKIP**
* **INFO**
* **PASS**

Please fix the descriptions and we're good.

@davelab6 if you're happy with the following rendering results, we can push once the description is fixed.

Desktop_Windows_10_chrome_69 0_

Desktop_Windows_10_chrome_69 0_

We have some cap height jumps. It may be better to push Kreon first because its vf has similar jumps but less users so we can gauge the impact at a smaller scale.

@bghryct for the upstreams, please pr your changes back to the original upstream repo.

File: ofl/quicksand/DESCRIPTION.en_us.html (hunk 1 above)
Comment (bghryct) [reply ofl/quicksand/DESCRIPTION.en_us.html]: Updated. Thanks for the help Marc!

**Comment by m4rc1e** [comment]
## Fontbakery report

Fontbakery version: 0.7.8

<details>
<summary><b>[5] Quicksand-VF.ttf</b></summary>
<details>
<summary>🔥 <b>FAIL:</b> Checking file is named canonically.</summary>

* [com.google.fonts/check/canonical_filename](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/canonical_filename)
* 🔥 **FAIL** The file 'Quicksand-VF.ttf' must be renamed to 'QuicksandLight[wght].ttf' according to the Google Fonts naming policy for variable fonts.

</details>
<details>
<summary>⚠ <b>WARN:</b> Stricter unitsPerEm criteria for Google Fonts. </summary>

* [com.google.fonts/check/unitsperem_strict](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/unitsperem_strict)
* ⚠ **WARN** Even though unitsPerEm (1000) in this font is reasonable. It is strongly advised to consider changing it to 2000, since it will likely improve the quality of Variable Fonts by avoiding excessive rounding of coordinates on interpolations.

</details>
<details>
<summary>⚠ <b>WARN:</b> Are there caret positions declared for every ligature?</summary>

* [com.google.fonts/check/ligature_carets](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/ligature_carets)
* ⚠ **WARN** This font lacks caret position values for ligature glyphs on its GDEF table. [code: lacks-caret-pos]

</details>
<details>
<summary>⚠ <b>WARN:</b> Is there kerning info for non-ligated sequences?</summary>

* [com.google.fonts/check/kerning_for_non_ligated_sequences](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/kerning_for_non_ligated_sequences)
* ⚠ **WARN** GPOS table lacks kerning info for the following non-ligated sequences:
	- f + f
	- f + i
	- i + f
	- f + l
	- l + f
	- i + l

   [code: lacks-kern-info]

</details>
<details>
<summary>⚠ <b>WARN:</b> Name table strings must not contain the string 'Reserved Font Name'.</summary>

* [com.google.fonts/check/name/rfn](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/name.html#com.google.fonts/check/name/rfn)
* ⚠ **WARN** Name table entry ("Copyright 2019 The Quicksand Project Authors (https://github.com/andrew-paglinawan/QuicksandFamily.git), with Reserved Font Name "Quicksand"") contains "Reserved Font Name". This is an error except in a few specific rare cases.

</details>
<br>
</details>

### Summary

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | ℹ INFO | 🍞 PASS |
|:-----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 1 | 4 | 49 | 8 | 99 |
| 0% | 1% | 2% | 30% | 5% | 61% |

**Note:** The following loglevels were omitted in this report:
* **SKIP**
* **INFO**
* **PASS**


## Diff images: [qa.zip](http://35.188.158.120/media/318d604f-e090-426f-9b07-bddc61e3b210/qa.zip)

**Comment by m4rc1e** [comment]
@bghryct I've fixed up your pr by renaming the ttf and squashing your commits.

@davelab6 these VF prs are on hold until we solve the ttfautohint issue.

**Pull request merged**